sound/soc/intel/boards/bridge_cs35l56.c

```diff
@@ -127,7 +127,7 @@ int bridge_cs35l56_add_sidecar(struct snd_soc_card *card,
 
 int bridge_cs35l56_spk_init(struct snd_soc_card *card,
 			    struct snd_soc_dai_link *dai_links,
-			    struct sof_sdw_codec_info *info,
+			    struct asoc_sdw_codec_info *info,
```

Comment:
Commit message: soc_sdw_codec_info -> asoc_sdw_codec_info and soc_sdw_dai_info -> asoc_sdw_dai_info as the code does.

Reply:
Will update the commit message